docs/integrations/gateways/aws-api-gateway.mdx

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 2
+title: AWS API Gateway x Permit
+---
+
+This document describes how to create an API Gateway that uses a Permit authorizer to authenticate requests.
+
+## Create a Permit Authorizer
+In order to create an Authorizer that uses Permit, you will need to create a Lambda function that uses Permit to authenticate requests.
+
+### Create a Lambda Function
```

Comment:
Add admonition that states users can add a lambda in any language supported by Permit SDKs

Reply:
done